.dependabot/config.yml

```diff
@@ -0,0 +1,62 @@
+version: 1
```

Comment:
@vexx32, your last commit had 10 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

Extended FileSystem Path/Location Cmdlet Provider Tests.Validate Resolve-Path Cmdlet Parameters.Verify relative
```powershell
Expected strings to be the same, but they were different.
Expected length: 2
Actual length:   39
Strings differ at index 1.
Expected: './'
But was:  '../3da3f0fb-48d9-4b4a-8cfb-3d27c92e73c2'
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1: line 1262
1262:             $relativePath | Should -BeExactly (Join-Path "." "")
```
Extended FileSystem Path/Location Cmdlet Provider Tests.Validate Join-Path Cmdlet Parameters.Validate Resolve
```powershell
Expected strings to be the same, but they were different.
Expected length: 19
Actual length:   50
Strings differ at index 0.
Expected: 'TestDrive:/Level1_0'
But was:  '/tmp/3da3f...'
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1: line 1273
1273:                 $result | Should -BeExactly "TestDrive:/$level1_0"
```
Extended FileSystem Path/Location Cmdlet Provider Tests.Valdiate Set-Location Cmdlets Parameters.By LiteralPath
```powershell
Expected strings to be the same, but they were different.
Expected length: 19
Actual length:   50
Strings differ at index 0.
Expected: 'TestDrive:/Level1_0'
But was:  '/tmp/3da3f...'
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1: line 1330
1330:             $result.Path | Should -BeExactly $level1_0Full
```
Extended FileSystem Path/Location Cmdlet Provider Tests.Valdiate Push-Location and Pop-Location Cmdlets Parameters.Verify Push + Path
```powershell
Expected strings to be the same, but they were different.
Expected length: 28
Actual length:   59
Strings differ at index 0.
Expected: 'TestDrive:/Level1_0/Level2_0'
But was:  '/tmp/3da3f...'
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1: line 1355
1355:             $location1.Path | Should -BeExactly $level2_0Full
```
Extended FileSystem Path/Location Cmdlet Provider Tests.Valdiate Push-Location and Pop-Location Cmdlets Parameters.Verify Push + PassThru
```powershell
Expected strings to be the same, but they were different.
Expected length: 28
Actual length:   59
Strings differ at index 0.
Expected: 'TestDrive:/Level1_0/Level2_0'
But was:  '/tmp/3da3f...'
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1: line 1371
1371:             $location3.Path | Should -BeExactly $level2_0Full
```